### ISSUE TITLE
Named placeholders

### ISSUE THREAD

**Issue by e-oz** [blackbeam/rust-mysql-simple#31]
If it's not too difficult, please consider to add named placeholders.
example:
```sql
SELECT  id FROM location WHERE
facility_name LIKE :search 
OR address LIKE :search
OR address2 LIKE :search
OR city LIKE :search
OR state LIKE :search
OR zip LIKE :search
OR rep_fname LIKE :search
```

**Comment by blackbeam** [comment]
I'm not sure that it's simple, but it's definitely a desired feature.
I'll try to find time for this.

**Comment by blackbeam** [comment]
Implemented in v5.0.0

**Comment by e-oz** [comment]
wow, cool! thank you!
